externs.js

```diff
@@ -568,5 +568,6 @@ var TopLevel = {
     "multiAccountDeriveAddresses" : function () {},
     "multiAccountReset" : function () {},
     "multiAccountLoadAccount" : function () {},
-    "multiAccountStoreAccount" : function () {}
+    "multiAccountStoreAccount" : function () {},
+    "multiAccountImportMnemonic" : function () {},
```

Comment:
@dmitryn  something wrong with methods, here you have 2 , but in java you have `multiAccountDeriveAddresses` in ios only `multiAccountImportMnemonic`

Reply:
@flexsurfer maybe it was merge conflicts, now i'm only adding `multiAccountImportMnemonic` to both java and objective-c code.